docs/user/communication.rst

```diff
@@ -41,99 +45,196 @@ Exemplary exercise detail page with posts:
 |exercise-posts|
 
 Course-wide Posts & Course Communication Overview
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+-------------------------------------------------
 
-Besides lecture or exercise related questions, Artemis offers a third post type: posts with course-wide topics, such as ``Organization`` or ``Tech Support``.
-These posts can only be created on the course communication overview, which is shown in the screenshot below.
+Besides lecture or exercise related questions, Artemis offers a third post type: posts with course-wide topics, such as
+``Organization`` or ``Tech Support``. These posts can only be created on the course communication overview, which is shown
+in the screenshot below.
 
 The ``Communication`` space of an Artemis course serves as overview for *all* posts in a course.
 Hence, course-wide posts as well as exercise posts and lecture posts are listed.
 Here, users can easily query, sort, and filter existing posts.
 
 |course-posts|
 
+Secluded Communication
```

Comment:
```suggestion
Private Communication
```
```suggestion
Direct Communication
```

Secluded sounds like the wrong word to me here (e.g., a house in the middle of nowhere is secluded).

Reply:
Replaced with 'Private'.